src/app/components/Toolbar/Features/Account/ManageableAccountDetails.tsx

```diff
@@ -0,0 +1,162 @@
+import { useContext, useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import copy from 'copy-to-clipboard'
+import { Box } from 'grommet/es6/components/Box'
+import { Button } from 'grommet/es6/components/Button'
+import { Form } from 'grommet/es6/components/Form'
+import { Notification } from 'grommet/es6/components/Notification'
+import { ResponsiveContext } from 'grommet/es6/contexts/ResponsiveContext'
+import { Tab } from 'grommet/es6/components/Tab'
+import { Tabs } from 'grommet/es6/components/Tabs'
+import { Text } from 'grommet/es6/components/Text'
+import { Copy } from 'grommet-icons/es6/icons/Copy'
+import { useTranslation } from 'react-i18next'
+import { NoTranslate } from 'app/components/NoTranslate'
+import { PasswordField } from 'app/components/PasswordField'
+import { persistActions } from 'app/state/persist'
+import { selectEnteredWrongPassword, selectPasswordCheckPass } from 'app/state/persist/selectors'
+import { preventSavingInputsToUserData } from 'app/lib/preventSavingInputsToUserData'
+import { Wallet } from '../../../../state/wallet/types'
+import { DerivationFormatter } from './DerivationFormatter'
+import { AddressBox } from '../../../AddressBox'
+import { layerOverlayMinHeight } from '../layer'
+import { LayerContainer } from './../LayerContainer'
+import { uintToBase64, hex2uint } from '../../../../lib/helpers'
+
+interface ManageableAccountDetailsProps {
+  wallet: Wallet
+}
+
+export const ManageableAccountDetails = ({ wallet }: ManageableAccountDetailsProps) => {
+  const { t } = useTranslation()
+  const dispatch = useDispatch()
+  const enteredWrongPassword = useSelector(selectEnteredWrongPassword)
+  const passwordCheckPass = useSelector(selectPasswordCheckPass)
+  const [layerVisibility, setLayerVisibility] = useState(false)
+  const [password, setPassword] = useState('')
+  const [acknowledge, setAcknowledge] = useState(false)
+  const [notificationVisible, setNotificationVisible] = useState(false)
+  const isMobile = useContext(ResponsiveContext) === 'small'
+  const copyAddress = () => {
+    const wasCopied = copy(uintToBase64(hex2uint(wallet.privateKey!)))
+    if (wasCopied) {
+      setNotificationVisible(true)
+    }
+  }
+  const hideLayer = () => {
+    dispatch(persistActions.resetPasswordCheckPass())
+    dispatch(persistActions.resetWrongPassword())
+    setPassword('')
+    setAcknowledge(false)
+    setLayerVisibility(false)
+  }
+  const onSubmit = () => dispatch(persistActions.checkPasswordAsync({ currentPassword: password }))
```

Comment:
Error if user without a profile wants to view their private key

Reply:
unpersisted has it's own flow now